src/elements/play-preview.ts

```diff
@@ -149,10 +149,11 @@ export class PlayPreview extends LitElement {
       if (this.bundle) this.#meta['actor-id']!.values = [this.bundle.hostname]
       this.dispatchEvent(Bubble<undefined>('clear-errors', undefined))
     }
+
     if (props.has('remoteRuntimeOrigin'))
-      this.#remote = RemoteApp.new(
-        this.remoteRuntimeOrigin,
-        () => this.uploaded ?? Promise.resolve({})
```

Comment:
Wasn't sure what this line was. Let me know if this is something really important we need to add to the actual RemoteApp interface in ui-renderer